src/prediction.jl

```diff
@@ -0,0 +1,252 @@
+    "Ruminococcus_sp_CAG_488", "Ruminococcus_sp_CAG_624", "Ruminococcus_torques", "Ruthenibacterium_lactatiformans", # R - 29 species
+    
+    "Saccharomyces_cerevisiae", "Salmonella_enterica", "Sanguibacteroides_justesenii", "Scardovia_wiggsiae", "Sellimonas_intestinalis",
+    "Serratia_liquefaciens", "Serratia_marcescens", "Slackia_isoflavoniconvertens", "Sneathia_amnii", "Solobacterium_moorei",
+    "Staphylococcus_argenteus", "Staphylococcus_aureus", "Staphylococcus_epidermidis", "Staphylococcus_hominis", "Staphylococcus_lugdunensis",
+    "Stenotrophomonas_maltophilia", "Streptococcus_agalactiae", "Streptococcus_anginosus_group", "Streptococcus_australis", "Streptococcus_gallolyticus",
+    "Streptococcus_gordonii", "Streptococcus_infantarius", "Streptococcus_infantis", "Streptococcus_lutetiensis", "Streptococcus_milleri",
+    "Streptococcus_mitis", "Streptococcus_mutans", "Streptococcus_oralis", "Streptococcus_parasanguinis", "Streptococcus_parauberis",
+    "Streptococcus_pasteurianus", "Streptococcus_peroris", "Streptococcus_pneumoniae", "Streptococcus_pseudopneumoniae", "Streptococcus_salivarius",
+    "Streptococcus_salivarius_CAG_79", "Streptococcus_sanguinis", "Streptococcus_sobrinus", "Streptococcus_sp_A12", "Streptococcus_sp_F0442",
+    "Streptococcus_sp_HPH0090", "Streptococcus_sp_M334", "Streptococcus_sp_SK643", "Streptococcus_thermophilus", "Streptococcus_vestibularis",
+    "Succinatimonas_hippei", "Sutterella_parvirubra", # S - 47 species
+    
+    "Tannerella_forsythia", "Terrisporobacter_othiniensis", "Tissierellia_bacterium_KA00581", "Treponema_succinifaciens", "Trueperella_bernardiae",
+    "Turicibacter_sanguinis", "Turicimonas_muris", "Tyzzerella_nexilis", # T - 8 species
+    
+    "Ureaplasma_parvum", # U - 1 species
+    
+    "Varibaculum_cambriense", "Veillonella_atypica", "Veillonella_dispar", "Veillonella_infantium", "Veillonella_parvula",
+    "Veillonella_rodentium", "Veillonella_rogosae", "Veillonella_seminalis", "Veillonella_sp_CAG_933", "Veillonella_sp_T11011_6",
+    "Veillonella_tobetsuensis", "Veillonellaceae_bacterium_DNF00626", "Victivallis_vadensis", # V - 13 species
+    
+    "Weissella_cibaria", "Weissella_confusa", # W - 2 species
+]
+
+function fix_metadata_colnames!(longdata_df::DataFrame)
+
+    ## This function was created to fix small inconsistencies and typos on the longdata table.
+    ## As original data cleanup and wrangling progresses, it will be altered and ultimately deprecated.
+    ## Comments are provided to help review transformations
+
+    # 1. Rename "[Collinsella]_massiliensis" to "Collinsella_massiliensis"
+    longdata_df[longdata_df.variable .== "[Collinsella]_massiliensis", :variable] .= "Collinsella_massiliensis"
+
+    return(longdata_df)
+    
+end
+
+function check_longdata_metaduplicates!(longdata_df::DataFrame; remove_duplicates=true)
+
```

Comment:
Indent is already giving visual separation, I don't think adding newline is useful here.

Reply:
I mean instead of 

```
function check_longdata_metaduplicates!(longdata_df::DataFrame; remove_duplicates=true)

    n_unique_rows = size(unique(longdata_df[:, 1:3]),1)    
    
    if (n_unique_rows != size(longdata_df, 1))

        n_nonunique = size(longdata_df, 1) - n_unique_rows

        if remove_duplicates    

            @warn "Long Dataframe contains non-unique rows! Argument `remove_duplicates` set to `true`. Removing duplicated data."
            @warn "After removal, $(n_unique_rows) will remain. $(n_nonunique) rows were rmoved from the original $(size(longdata_df, 1))"
            rawDf = unique!(longdata_df)
            return(longdata_df)

        else

            @warn "Long Dataframe contains non-unique rows! Argument `remove_duplicates` set to `false`. Returning source data."
            return(longdata_df)

        end #end if remove_duplicates

    end # end if n_unique_rows
    
end # end function
``` 

I prefer something like

```

function check_longdata_metaduplicates!(longdata_df::DataFrame; remove_duplicates=true)
    n_unique_rows = size(unique(longdata_df[:, 1:3]),1)    
    
    if (n_unique_rows != size(longdata_df, 1))
        n_nonunique = size(longdata_df, 1) - n_unique_rows

        if remove_duplicates    
            @warn "Long Dataframe contains non-unique rows! Argument `remove_duplicates` set to `true`. Removing duplicated data."
            @warn "After removal, $(n_unique_rows) will remain. $(n_nonunique) rows were rmoved from the original $(size(longdata_df, 1))"
            rawDf = unique!(longdata_df)
            return(longdata_df)
        else
            @warn "Long Dataframe contains non-unique rows! Argument `remove_duplicates` set to `false`. Returning source data."
            return(longdata_df)
        end #end if remove_duplicates
    end # end if n_unique_rows
end # end function
```

Obviously, such things are mostly aesthetic, but :shrug: